setup.cfg

```diff
@@ -81,7 +84,9 @@ universal = 1
 testpaths =
     tests
 junit_family = xunit2
-addopts = '-Wd'
+addopts = --benchmark-disable -ra --showlocals --strict-config
```

Comment:
So `--strict-config` is the only providing the failure in Python 2.7. I wonder if `addopts` can be dependent on the Python version as dependencies specified in this file?

Reply:
We will just have to leave it off until we drop 2.7.

Reply:
(It's stuck with an old PyTest)